apps/ledger-live-mobile/package.json

```diff
@@ -145,6 +145,7 @@
     "expo-keep-awake": "~12.8.2",
     "expo-modules-autolinking": "^1.10.2",
     "expo-modules-core": "^1.11.8",
+    "expo-random": "^13.6.0",
```

Comment:
@KVNLS It is required for this package : https://www.npmjs.com/package/@ton/crypto-primitives

@ayelenmurano do you think there is a way to not use it / use a different lib ?

Reply:
(original pr : https://github.com/LedgerHQ/ledger-live/pull/6983)

Reply:
@hedi-edelbloute @KVNLS I don't think there is an easy way to get rid of this, as it is used (at the end) by @ton/core dependency, which is the main package we use for the ton integration. Expo-random is really buried on the dependency graph.

Reply:
Can we use expo-random `12.3.0` instead of `13.6.0`, please? Because it's already declared as a dependency of hashgraph we could avoid having two different versions of this lib.
Ok to merge with expo-random but I think it should be the responsibility of the client app to choose the crypto polyfill they use.

Reply:
@emmanuelm41, for a long-term solution, could you reconsider depending on `expo-random` directly here https://github.com/ton-org/ton-crypto-primitives/blob/f5c7482bf8d03ea39fa1668ec0fe458e0b516658/src/native/getSecureRandom.ts#L9, please? This would help applications choose the polyfill they use for crypto

Reply:
@KVNLS I can try to use 12.3.0 instead of 13.6.0, but in the past we needed to downgrade from 14.1.0 to 13.6.0 because the integration was failing. Three cases is a major change on the version, so that indicates changes should not be compatible between them.

Reply:
> @emmanuelm41, for a long-term solution, could you reconsider depending on `expo-random` directly here https://github.com/ton-org/ton-crypto-primitives/blob/f5c7482bf8d03ea39fa1668ec0fe458e0b516658/src/native/getSecureRandom.ts#L9, please? This would help applications choose the polyfill they use for crypto

Regarding this, I am not 100% what you mean. Can you explain me more?

Reply:
Maybe you can replace
```
const getRandomBytes = require('expo-random').getRandomBytes;

export function getSecureRandomBytes(size: number): Buffer {
    return Buffer.from(getRandomBytes(size));
}
```
By
```
const crypto = require('crypto')
export function getSecureRandomBytes(size: number): Buffer {
    return Buffer.from(crypto.getRandomValues(size));
}
```

In that case, it's the responsibility of the React Native application to provide the `crypto` module polyfill.
Btw this is what you do with `Buffer`.

Reply:
We don't have control over that package actually. It is on the ton foundation. I have asked if there is anything we can do quickly. I am not quite sure how fast this can be addressed though.

Reply:
It looks like the build fails with version 12 of expo random

https://github.com/LedgerHQ/ledger-live/pull/7405/commits/ffa9a0e7ba5be8e052bf113697eb675c4ed34d8f
https://github.com/LedgerHQ/ledger-live-build/actions/runs/10096295649

Reply:
Are you ok with us using 13.6 ? long term hopefully we have a fix in the underlying lib

Reply:
If it's possible we can try on develop to use 13.6 on hedera

Reply:
@KVNLS @hedi-edelbloute Here you have the PR to remove expo-random

https://github.com/LedgerHQ/ledger-live/pull/7436

Reply:
Build fails
![image](https://github.com/user-attachments/assets/9358f93e-d3bb-45b1-907a-b3dd1a3f6b1b)

Reply:
The build should work now 
https://github.com/LedgerHQ/ledger-live/pull/7436